include/EUTelGeometryTelescopeGeoDescription.h

```diff
@@ -23,6 +23,14 @@
 // EUTELESCOPE
 #include "EUTelUtility.h"
 #include "EUTelGenericPixGeoMgr.h"
+#include "EUTelState.h"
+
+//LCIO
+#include "lcio.h"
+#include "IMPL/TrackerHitImpl.h"
+#include "IMPL/TrackImpl.h"
+#include <UTIL/LCTOOLS.h>
```

Comment:
This should go away, any functionality which requires LCIO must be in an own class